pyproject.toml

```diff
@@ -69,6 +73,9 @@ installer = "pip"
 parallel = true
 randomize = true
 extra-args = ["-m", "not integration"]
+extra-dependencies = [
+    "PyKCS11",
+]
```

Comment:
can we declare the need for `PyKCS11` as a feature?
https://hatch.pypa.io/latest/config/environment/overview/#features
```toml
features = [
  "pkcs11",
]
```